### ISSUE TITLE
Spring Insight for Tomcat

### ISSUE THREAD

**Issue by anujgandharv** [spring-projects/spring-insight-plugins#1]
Not sure whether this query belongs here or not. If it doesnt, do let me know and I can move it to the right place.
Does anyone know how I can integrate Spring Insight with Tomcat 6? I have been trying to search for documentation throughout but cant find any. Any help would be really appreciated.

Thanks,

**Comment by StefanieLover** [comment]
I set up my liferay followed the instructions mentioned above.Unfortunately tomcat was not able to find the classes. Can anyone provide a logical solution on Integrated tomcat with spring insight?

**Comment by bkiran** [comment]
HI

           I have set up the application long back spring with tomcat if
you don't mind will provide that code base to you by tomorrow, May be it
will helpful to you.

Thanks,
Kiran B.

**Comment by StefanieLover** [comment]
HI @anchlovi  Have you successed by following the instructions you provide? Imeet ClassNotFoundException.

**Comment by bkiran** [comment]
Yes succeed based on the @anchlovi <https://github.com/anchlovi>
 suggesstions
